plugins/examples/host-toolchain/pkgconf-host.mk

```diff
@@ -17,7 +17,8 @@ endef
 define $(PKG)_BUILD
     cd '$(SOURCE_DIR)' && ./autogen.sh
     cd '$(BUILD_DIR)' && '$(SOURCE_DIR)/configure' \
-        $(MXE_CONFIGURE_OPTS)
+        $(MXE_CONFIGURE_OPTS) \
+        CFLAGS="$(CFLAGS) $(if $(BUILD_STATIC),-DPKGCONFIG_IS_STATIC)"
```

Comment:
I think `$(CFLAGS)` is a `make` variable in this context and likely to be empty

Reply:
Thanks, then perhaps it is not needed even in cairo.mk?
https://github.com/mxe/mxe/commit/c950f3635caa9b2dd562286baa51138e6e30dca1